name-service/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.openshift.booster</groupId>
+    <artifactId>wfswarm-cache-booster</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>wfswarm-cache-name</artifactId>
+  <packaging>war</packaging>
+
+  <name>WildFly Swarm - Cache Booster - Name Service</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax</groupId>
+      <artifactId>javaee-api</artifactId>
```

Comment:
Again, I think this isn't needed if you depend on the individual fractions directly.

Reply:
removed